projects/core/src/configurator/commons/connectors/configurator-commons.adapter.ts

```diff
@@ -53,4 +53,15 @@ export abstract class ConfiguratorCommonsAdapter {
   abstract readPriceSummary(
     configuration: Configurator.Configuration
   ): Observable<Configurator.Configuration>;
+
+  /**
+   * Abstract method to get configuration overview
+   *
+   * @param configId configuration id
+   * @param owner configuration owner
+   */
+  abstract getConfigurationOverview(
+    configId: string,
+    owner: Configurator.Owner
+  ): Observable<Configurator.Configuration>;
```

Comment:
Should return Observable<Configurator.Overview>

Reply:
Done.